src/graphql/mirror.js

```diff
@@ -35,6 +36,39 @@ export class Mirror {
     this._initialize();
   }
 
+  /**
+   * Embed the GraphQL schema into the database, initializing it for use
+   * as a mirror.
+   *
+   * We store the data for an object across three tables, depending on
+   * the field type:
+   *
+   *   - Connections are stored in the `connections `table.
+   *   - Node references are stored in the `links` table.
+   *   - Primitive data is stored in a type-specific data table.
+   *
+   * We refer to node and primitive data together as "own data", because
+   * this is the data that can be queried uniformly for all elements of
+   * a type; querying connection data, in contrast, requires the
+   * object-specific end cursor.
+   *
+   * The aforementioned tables are each keyed by object ID. An object's
+   * metadata appears in the `objects` table, including the object's
+   * typename and the last update for the object's own-data. Each
+   * connection has its own last update value, because connections can
+   * be updated independently of each other and of own-data.
+   *
+   * Note that any object in the database should have entries in the
+   * `connections` and `links` table even if the node has never been
+   * updated.
```

Comment:
I'm a little confused how this is the case. It would be great to have a little example of the lifecycle and how/why we maintain this property